addons/payment_authorize/models/payment_transaction.py

```diff
@@ -188,7 +197,8 @@ def _process_feedback_data(self, data):
                 self._set_authorized()
                 if self.tokenize and not self.token_id:
                     self._authorize_tokenize()
-                self._send_refund_request(create_refund_transaction=False)  # In last step because it calls _handle_feedback_data()
+                if self.operation == 'validation':
+                    self._send_refund_request(create_refund_transaction=False)  # In last step because it calls _handle_feedback_data()
```

Comment:
```suggestion
                    # Void the transaction. In last step because it calls _handle_feedback_data().
                    self._send_refund_request(create_refund_transaction=False)
```